homeassistant/components/renault/quality_scale.yaml

```diff
@@ -0,0 +1,64 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
Tests are not asserting the unique id

Reply:
> Tests are not asserting the unique id

Thanks for pointing out - I've adjusted the scale accordingly